src/apps/tools/OswAppWebserver.cpp

```diff
@@ -22,7 +22,10 @@ void OswAppWebserver::loop() {
     if (OswServiceAllTasks::wifi.isConnected()) {
         hal->gfx()->print(LANG_DISCONNECT);
     } else {
-        hal->gfx()->print(LANG_CONNECT);
+        if(OswServiceAllTasks::wifi.isEnabled())
+            hal->gfx()->print("...");
```

Comment:
What is mean?

Reply:
If so, you should also change the part where you `time_config`.

Reply:
Show `...` while we connect, but are not yet connected.

> If so, you should also change the part where you time_config.

Good catch!